app/views/transactions/index.njk

```diff
@@ -41,11 +41,23 @@
 <div class="govuk-grid-column-full">
   <h1 class="govuk-heading-l">
     {% if allServiceTransactions %}
-       Transactions for all live services
+       Transactions for all services
+       <strong class="govuk-tag govuk-tag--grey">{{ "LIVE" if filterLiveAccounts else "TEST" }}</strong>
     {% else %}
        Transactions
     {% endif %}
   </h1>
+
+  {% if allServiceTransactions %}
+    {% if filterLiveAccounts %}
+      <p><a class="govuk-link govuk-link--no-visited-state" href="{{ routes.formattedPathFor(routes.allServiceTransactions.indexStatusFilter, 'test') }}">Switch to test</a></p>
```

Comment:
Can we say `Switch to test accounts`? Content updated (final now) this afternoon.

Similarly `Switch to live accounts` below